### PR TITLE
fix(i18n): update en.php

### PR DESCRIPTION
**Pull request by uidHUB**
Remove double line `messages:inbox` in `mod/messages/languages/en.php`

### CODE REVIEW

File: mod/messages/languages/en.php [hunk 1]
```diff
@@ -15,7 +15,6 @@
 	'messages:title' => "Subject",
 	'messages:to:help' => "Write recipient's username here.",
 	'messages:replying' => "Message replying to",
-	'messages:inbox' => "Inbox",
```
Comment (hypeJunction) [inline mod/messages/languages/en.php]: If you look 6 lines above, you will see that it's there already. https://github.com/Elgg/Elgg/pull/9847/files#diff-a884bee3e6e6d9e615975c1a12bca57cL12

**Comment by hypeJunction** [comment]
Thanks for the contribution.

We have a commit message policy that is enforced by CI tests. Next time your contribute, please take a look at http://learn.elgg.org/en/2.0/contribute/code.html#commit-message-format

This change is unnecessary, because the string is already there.

**Comment by KitsuneSolar** [comment]
I deleted that line, because it is repeated.

https://github.com/Elgg/Elgg/blob/2.x/mod/messages/languages/en.php#L12-L18

Line 12 & 18.

**Comment by hypeJunction** [comment]
Oh, sorry, I am just looking the wrong the way. 
We could accept the fix if you update the commit message.

**Comment by KitsuneSolar** [comment]
@hypeJunction, please check. :)

**Comment by hypeJunction** [comment]
You need to update the commit message, not the pull request title.
https://help.github.com/articles/changing-a-commit-message/

I don't think we want in this in changelogs, so a more appropriate message would be:
`chore(messages): removes duplicate translation string`

**Comment by mrclay** [comment]
Please submit the commit to the 1.12 branch instead, as the duplicate appears there, too.

**Comment by KitsuneSolar** [comment]
Guys, this is too difficult for me. :(
I can hardly working with Git.

Let me just point out the error? And you fix it.

**Comment by beck24** [comment]
Yes that's fine, thank you for pointing it out

**Pull request closed without merging**